documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentActionMenu.vue

```diff
@@ -29,23 +29,15 @@ export default {
     menuExtensionApp: 'DocumentMenu',
     menuExtensionType: 'menuActionMenu',
     menuExtensions: {},
-    mobileOnlyExtensions: ['favorite','details'],
-    desktopOnlyExtensions: ['edit'],
     editExtensions: ['edit'],
-    fileOnlyExtension: ['download','favorite'],
-    redactorExtension: ['download', 'versionHistory','details', 'copyLink'],
```

Comment:
why did you remove mobileOnly, destopOnly and fileOnly filters ?

We only wnat to remove redatorExtension

Reply:
I moved the check to the enabled function of the extension so we will not loop in extensions on each filter.
I fact i have reworked somthings + removing the redactor filter